spring-integration-core/src/main/java/org/springframework/integration/channel/NullChannel.java

```diff
@@ -27,32 +32,144 @@
  * All receive() calls will return <em>null</em>, and all send() calls
  * will return <em>true</em> although no action is performed.
  * Note however that the invocations are logged at debug-level.
- * 
+ *
  * @author Mark Fisher
+ * @author Gary Russell
  */
-public class NullChannel implements PollableChannel {
+@ManagedResource
+public class NullChannel implements PollableChannel, MessageChannelMetrics, NamedComponent {
```

Comment:
It's unlikely, I suppose, but this way you could replace any channel in the context with a `NullChannel` and the context would still load/run (although, of course, there would be no messages downstream :smile: ).